Enterprise-CMCS/macpro-mdct-mcr
# [MDCT-2662]: Trigger validation when coming from Review and Submit

### Description


Validation wouldn't trigger when navigating from the Review and Submit page to a page the user hadn't actually visited before in their current session. This changes it so that now if a user clicks the edit button on Review and Submit, all fields that don't meet validation will be triggered regardless if they've ever been on the page before!

https://github.com/Enterprise-CMCS/macpro-mdct-mcr/assets/19439679/e2247ebe-2a2f-46db-a9da-51057ff9ae15



### Related ticket(s)


MDCT-2662

---
### How to test

Create a report
Go to Review and Submit
Click on any page! It should trigger validation across all pages, including special pages like Modal Drawers, Drawer Reports, Standard Report pages, and more!

---
### Author checklist
